.github/workflows/update-helm-charts-index.yml

```diff
@@ -0,0 +1,53 @@
+name: hashicorp/vault-helm/update-helm-charts-index
+on:
+  push:
+    tags:
+      - 'v[0-9]+\.[0-9]+\.[0-9]+\.*'
+
+jobs:
+  update-helm-charts-index:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: verify Chart version matches tag version
+        run: |-
+          export TAG=${RELEASE_TAG:-${{ github.ref_name }}}
+          git_tag=$(echo "${TAG#v}")
+          chart_tag=$(yq r Chart.yaml version)
+          if [ "${git_tag}" != "${chart_tag}" ]; then
+            echo "chart version (${chart_tag}) did not match git version (${git_tag})"
+            exit 1
+          fi
+        env:
+          RELEASE_TAG: "${{ inputs.release-tag }}"
+      - name: update helm-charts index
+        id: update
+        run: |-
+          export GITHUB_TOKEN="${{secrets.helm_charts_github_token }}"
+          gh workflow run publish-charts.yml \
+            --repo hashicorp/helm-charts \
+            --ref main \
+            -f SOURCE_TAG="${{ github.ref_name }}" \
+            -f SOURCE_REPO="${{ github.repository }}"
+      - name: Prepare Slack Notification
+        if: always()
+        run: |
+          case "${{ steps.update.conclusion }}" in
+            failure)
+              echo "SLACK_MESSAGE=:red_circle: The vault-helm charts index update failed." >> "$GITHUB_ENV"
+              echo "SLACK_BLOCK_COLOR=ed5c5c" >> "$GITHUB_ENV"
+              ;;
+            success)
+              echo "SLACK_MESSAGE=:tada: The vault-helm charts index update succeeded." >> "$GITHUB_ENV"
+              echo "SLACK_BLOCK_COLOR=1cbf43" >> "$GITHUB_ENV"
+              ;;
+          esac
+      - name: Send Slack Notification
+        # note: this will skip Slack notification if the job is skipped or cancelled.
+        if: ${{ (success() || failure()) && env.SLACK_MESSAGE != '' }}
+        uses: ./.github/actions/slack-status
+        with:
+          block-color: ${{ env.SLACK_BLOCK_COLOR }}
+          message: ${{ env.SLACK_MESSAGE }}
+          run-url: ${{ env.RUN_URL }}
+          slack-webhook-url: ${{ secrets.slack_webhook_url }}
```

Comment:
I named the secret `SLACK_WEBHOOK` as per the PR description, but looks like this is expecting a different name?

Reply:
Yeah, that's my mistake. I'll correct the capitalization. They aren't POSIX vars anymore, so that's just a habit of mine.